OpenQA/Isotovideo/Runner.pm

```diff
@@ -3,6 +3,10 @@
 
 package OpenQA::Isotovideo::Runner;
 use Mojo::Base -base, -signatures;
+# Avoid "Subroutine JSON::PP::Boolean::(0+ redefined" warnings
+# Details: https://progress.opensuse.org/issues/90371
+use JSON::PP;
```

Comment:
Have you checked whether we really need this here as well now?

Reply:
yes. it doesnt work without it. See also https://github.com/os-autoinst/os-autoinst/pull/2474#discussion_r1540974446